build fails for MIPS architecture

podman 2.2.1 fails to build on MIPS (both little and big endian, 32-bit as well as 64-bit) due to some type incompatibility:
```
# github.com/containers/psgo/internal/dev
../../../../../dl/go-mod-cache/github.com/containers/psgo@v1.5.1/internal/dev/tty.go:116:22: cannot use s.Rdev (type uint32) as type uint64 in argument to minDevNum
../../../../../dl/go-mod-cache/github.com/containers/psgo@v1.5.1/internal/dev/tty.go:117:22: cannot use s.Rdev (type uint32) as type uint64 in argument to majDevNum
```

Please see the links below for full build logs:

https://downloads.openwrt.org/snapshots/faillogs/mips64_octeonplus/packages/podman/compile.txt

https://downloads.openwrt.org/snapshots/faillogs/mips_24kc/packages/podman/compile.txt

https://downloads.openwrt.org/snapshots/faillogs/mipsel_24kc/packages/podman/compile.txt

--- a/internal/dev/tty.go
+++ b/internal/dev/tty.go
@@ -113,8 +113,9 @@ func TTYs() (*[]TTY, error) {
 		}
 		s := fi.Sys().(*syscall.Stat_t)
 		t := TTY{
-			Minor: minDevNum(s.Rdev),
-			Major: majDevNum(s.Rdev),
+			// Rdev is type uint32 on mips arch so we have to cast to uint64
+			Minor: minDevNum(uint64(s.Rdev)),
+			Major: majDevNum(uint64(s.Rdev)),
 			Path:  dev,
 		}
 		ttys = append(ttys, t)

